tests/custom.rs

```diff
@@ -16,7 +20,6 @@ impl AuthBasicCustom for MyCustomBasic {
     }
 }
 
-#[async_trait]
```

Comment:
This depends on the MSRV of the crate, I would guess.

Reply:
I think that Axum dropped support so this was not compatible. Is there a way to make it compatible for both versions?